examples/vg-specs/layer_text_heatmap.vg.json

```diff
@@ -225,6 +225,9 @@
                         "angle": {
                             "value": 270
                         },
+                        "align": {
+                            "value": "right"
+                        },
```

Comment:
I'm a little confused about where this addition came from as there's no associated edit to a `vl.json` spec?